ui/src/app/cohort-search/node-info/node-info.component.css

```diff
@@ -52,3 +52,9 @@ selection-icon-blue {
   white-space: nowrap;
   align-self: baseline;
 }
+
+/*.search-keyword {*/
+  /*color: #659F3D;*/
+  /*font-weight: bolder;*/
+  /*background-color: rgba(101,159,61,0.2);*/
+/*}*/
```

Comment:
Can we remove this since it's commented out?

Reply:
Done. Also removed the unused class from the html file